services/user-service/.env.example

```
@@ -0,0 +1,24 @@
+JWT_PUBLIC_KEY_FILE=/run/secrets/public.pem
+JWT_PRIVATE_KEY=
+JWT_PUBLIC_KEY=
+JWT_ALGORITHM=RS256
+ACCESS_TOKEN_EXPIRE_MINUTES=15
+REFRESH_TOKEN_EXPIRE_DAYS=7
+
+# OpenTelemetry
+OTEL_ENABLED=true
+OTEL_EXPORTER_OTLP_ENDPOINT=http://otel-collector:4317
+OTEL_EXPORTER_OTLP_INSECURE=true
+OTEL_PYTHON_EXCLUDED_URLS="health"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)